.github/actions/update-config/action.yml

```
@@ -34,21 +31,30 @@ runs:
         field: sdk.hostname
         value: "${{ inputs.host-name }}.mauth.nevis.cloud"
 
+    # Get SDK version number
+    - name: Get Version Number
+      id: get_sdk_version_number
+      shell: bash
+      run: |
+        SDK_VERSION=`cat pubspec.yaml | grep -o "nevis_mobile_authentication_sdk: '[^']*'" | cut -f2 -d"'" | xargs`
+        echo "SDK_VERSION=$(echo $SDK_VERSION)" >> $GITHUB_OUTPUT
```

🔧 Please keep the input/output naming convention.
```suggestion
        echo "sdk-version=$(echo $SDK_VERSION)" >> $GITHUB_OUTPUT
```